### ISSUE TITLE
SyntaxError: Failed to execute 'querySelectorAll' on 'Document'

### ISSUE THREAD

**Issue by petrbrzek** [wwayne/react-tooltip#119]
How you ever encounter this bug?

```
SyntaxError: Failed to execute 'querySelectorAll' on 'Document': '[data-tip][data-for="// Style for "Vector Sma" width: 27px; height: 37px;"]' is not a valid selector.
```

**Comment by petrbrzek** [comment]
Sorry, I'm seeing it right now.

**Comment by fzxt** [comment]
Sorry, how did you fix this? I'm having the same issues.

**Comment by wwayne** [comment]
@fzxt I released a new version 3.0.5 after checking the code, I don't know if it would solve your problem, if not, please give me more detail of your code.

**Comment by fzxt** [comment]
@wwayne Thanks for the quick response. Everything seems to be working fine now. I believe I was actually using data-id property incorrectly, had spaces in it. Once I removed the spaces from the id property everything was golden.
